package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "node-sass": "^4.11.0",
```

Comment:
Not sure what we want here, it is the default as per the CRA docs. Happy to change that :)